dash-pipeline/tests/libsai/vnet_out/vnet_out.cpp

```diff
@@ -3,8 +3,13 @@
 #include <unordered_map>
 #include <string.h>
 
+extern "C" {
 #include <sai.h>
+}
 
+// TODO below apis are extern of internal implementation
+// they should not be used by extern, but obtained via
+// sai_api_query function, FIXME
```

Comment:
Could you add an issue to ensure tracking? Thanks.

Reply:
sure

Reply:
done https://github.com/sonic-net/DASH/issues/414